onnxruntime/core/providers/openvino/backend_manager.cc

```diff
@@ -175,66 +209,58 @@ BackendManager::BackendManager(const GlobalContext& global_context,
 // precompiled blob is set. If that's the case:
 // By default, create model in embed mode where the blob stream is exported as data within
 // the EPContext node.
-Status BackendManager::ExportCompiledBlobAsEPCtxNode(const onnxruntime::GraphViewer& graph_body_viewer,
-                                                     const logging::Logger& logger) {
-  if (GetGlobalContext().disable_dynamic_shapes && subgraph_context_.has_dynamic_input_shape) {
+Status BackendManager::ExportCompiledBlobAsEPCtxNode(const onnxruntime::GraphViewer& graph_body_viewer) {
+  if (session_context_.disable_dynamic_shapes && subgraph_context_.has_dynamic_input_shape) {
     std::string exception_str =
         "Exporting dynamically compiled models at runtime is not supported. "
         "Cannot export blobs of dynamic models that request static shape inference. "
         "To export this model, set disable_dynamic_shapes to False";
     ORT_THROW(exception_str);
   }
 
```

Comment:
This was MSFT requirement to ensure that cache file dir and ep context model blob file path should be mapped ?
@preetha-intel  is there a functionality break here

Reply:
This check ensures that only epctx model or OV cache files will be generated. 
ep.context_enable takes the higher precedence